.vsts-ci/windows-daily.yml

```diff
@@ -6,6 +6,7 @@ trigger:
     include:
     - master
     - release*
+    - feature*
```

Comment:
Do we want daily builds for feature branches right now? The CI builds already run all the tests, should we start daily builds once the branch is more stable?

Reply:
We probably don't need daily builds on feature branches

Reply:
Will revert this one.

Reply:
@adityapatwardhan I still keep `feature*` in the `pr:` section of this file, does that look right?

Reply:
we do not use this yml for CI builds. we can remove it from `pr` section too to keep it consistent. when we have CI passing, i guess we will merget his branch to master.

Reply:
Sounds good. Will revert.

Reply:
@adityapatwardhan Done. Please take another look. Thanks!